packages/manager/src/features/CloudPulse/Widget/components/CloudPulseLineGraph.tsx

```diff
@@ -42,25 +32,14 @@ export const CloudPulseLineGraph = React.memo((props: CloudPulseLineGraph) => {
           <ErrorState errorText={error} />
         </Box>
       ) : (
-        <LineGraph
+        <AreaChart
           {...rest}
-          sxTableStyles={{
-            '& .MuiTable-root': {
-              border: 0,
-            },
-            backgroundColor: theme.bg.offWhite,
-            maxHeight: `calc(${theme.spacing(14)} + 3px)`,
-            minHeight: `calc(${theme.spacing(10)})`,
-            overflow: 'auto',
-            paddingLeft: theme.spacing(1),
-          }}
-          ariaLabel={ariaLabel}
-          data={data}
-          isLegendsFullSize={true}
-          legendRows={legendRows}
+          connectNulls
+          fillOpacity={0.5}
```

Comment:
```suggestion
          fillOpacity={theme.spacing(0.5)}
```

Reply:
I think direct value is fine to use as it does not have anything to do with theme. it'll just accept value between 0-1